config/locales/en.yml

```diff
@@ -27,6 +27,7 @@ en:
   deploy_button:
     hint:
       max_commits: It is recommended not to deploy more than %{maximum} commits at once.
+      blocked: This commit range includes a commit that can't be deployed.
```

Comment:
> can't be deployed *automatically* ?

Reply:
Nope, not deployed at all. It's akin to a build failure. You can use emergency mode if you want to ignore the status.